### PR TITLE
feat: jamie's updates to workflow and other stuff

### PR DESCRIPTION
**Pull request by jamiewaese-uncharted**
Added a semi-transparent glass effect to the Workflow toolbar.

Also added a new button, "Show all", which should zoom out and reposition the canvas to show everything. Function still needs to be written.

Monosnap Terarium 2023-06-01 10-53-31

### CODE REVIEW

**Review by dvince2 — APPROVED** [review]

File: packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue [hunk 1]
```diff
@@ -137,6 +143,7 @@ import AccordionTab from 'primevue/accordiontab';
 import Button from 'primevue/button';
 import { IProject, ProjectAssetTypes, ProjectPages, isProjectAssetTypes } from '@/types/Project';
 import { useDragEvent } from '@/services/drag-drop';
+import 'primeicons/primeicons.css';
```
Comment (mwdchang) [inline packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue:146]: This is imported at the top level of the application right? 

Why do we need a separate import here?

Comment (jamiewaese-uncharted) [reply packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue:146]: You're right.. it seems to work without it

File: packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue [hunk 2]
```diff
@@ -148,6 +155,7 @@ const props = defineProps<{
 
 const emit = defineEmits(['open-asset', 'open-overview', 'remove-asset', 'close-tab']);
 
+const hover = ref([] as boolean[]);
```
[the remark this answers is not in the document]
Comment (jamiewaese-uncharted) [reply packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue:158]: I didn't write that line. It seems to be related to all this stuff:

```
					@click="emit('open-asset', tab)"
					@mouseover="hover[tab?.assetId || 0] = true"
					@mouseleave="hover[tab?.assetId || 0] = false"
					@focus="hover[tab?.assetId || 0] = true"
					@focusout="hover[tab?.assetId || 0] = false"
					@dragstart="hover[tab?.assetId || 0] = false"
```

[the remark this answers is not in the document]
Comment (mwdchang) [reply packages/client/hmi-client/src/page/project/components/tera-resource-sidebar.vue]: simplified

**Pull request merged**